src/libraries/System.IO.FileSystem/tests/FileStream/ctor_options.cs

```diff
@@ -0,0 +1,172 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Linq;
+using System.Security.Cryptography;
+using Xunit;
+
+namespace System.IO.Tests
+{
+    // Don't run in parallel as the WhenDiskIsFullTheErrorMessageContainsAllDetails test
+    // consumes entire available free space on the disk (only on Linux, this is how posix_fallocate works)
```

Comment:
This comment is now stale, since we no longer use posix_fallocate, yes?

Reply:
It may still be true also when using `fallocate`.